packages/manager/apps/ips/src/data/api/postorput/organisations.ts

```diff
@@ -0,0 +1,15 @@
+import apiClient, { ApiResponse } from '@ovh-ux/manager-core-api';
+import { OrgDetails } from '@/data/api/get/organisationsDetails';
+
+export const postorputOrganisations = (
```

Comment:
```suggestion
export const upsertOrganisation = (
```
Seems more adapted, what do you think ?